k8s/base/4_deployment.yaml

```diff
@@ -35,7 +35,7 @@ spec:
       containers:
       - name: opentelemetry-collector
         imagePullPolicy: Always
-        image: otel/opentelemetry-collector-contrib:0.118.0
+        image: us-docker.pkg.dev/cloud-ops-agents-artifacts/google-cloud-opentelemetry-collector/otelcol-google:v0.121.0
```

Comment:
`ImagePullBackoff`: Image address needs to be checked.

Reply:
Needed to drop the `v` in the image tag